core/src/main/java/com/opentable/logging/CommonLogHolder.java

```diff
@@ -47,6 +47,10 @@
         }
         LOG.info("Detected hostname as '{}'", HOST_NAME);
 
+        // If not present, will be null, and not make it through buzzsaw.
+        INSTANCE_NO = System.getenv("INSTANCE_NO");
```

Comment:
Great suggestion.  Parsed out as `Integer` in f3f2d9aaf477c2b9beec7ecec02adf11f1edb007.